fixups.json

```diff
@@ -0,0 +1,9 @@
+{
+    "MQDebug": {
+        "0": {
+            "0x2a50": ["Envelope", 16],
+            "0x2a80": ["Envelope", 12],
+            "0x2a90": ["Envelope", 16]
+        }
+    }
+}
```

Comment:
Could this file use a more descriptive name? `audio_extraction_fixups.json` perhaps?

Reply:
This has since been resolved.